README.md

```diff
@@ -42,17 +42,17 @@ Actix web is a simple, pragmatic and extremely fast web framework for Rust.
 
 ## Docs
 
-* [API documentation (master)](https://actix.rs/actix-web/actix_web)
```

Comment:
This is the rendered version of master branch docs.

Reply:
Ok I've switched it.